examples/express-ts/src/api.ts

```diff
@@ -64,15 +64,15 @@ export default function (app: express.Application) {
       .catch(next);
```

Comment:
That was because you wrote `next(throwNotFound)` instead of `next(throwNotFound())`. Actually even the latter wouldn't work and it should have been something like `next(createNotFoundError())`. `res.sendStatus(404)` is fine though.